app/middlewares/validators/users.js

```diff
@@ -0,0 +1,28 @@
+const UserModel = require('../../models/user');
+const errors = require('../../errors');
+
+exports.checkExistingEmail = async email => {
+  const user = await UserModel.findOne({ where: { email } });
+
+  if (user) throw errors.badRequestError('User already exists');
+};
+
+exports.checkPasswordRestriction = password => {
+  const isAlphanumericRegex = /[^a-z\d]/i;
```

Comment:
In the regex, you are denying the letters and numbers, so here it would only accept special characters.